_episodes/11-using-arrays-transformations.md

```diff
@@ -12,42 +12,74 @@ keypoints:
+- an array containing a list of numbers could look like: [1, 2, 3, 4]
 
-"Monday,Tuesday,Wednesday,Thursday,Friday,Saturday,Sunday"
+Arrays can be sorted, de-duplicated, and manipulated in other ways in GREL expressions, but cannot be stored directly in an OpenRefine cell. Arrays in OpenRefine are usually the result of a transformation written with GREL. For example the ```split``` function takes a string, and changes it into an array based on a 'separator'. For example if a cell has the value:
 
-This can be transformed into an array using the ```split``` function
+"crystal structure|clozapinium|crystal structure|molecular configuration|hydrogen bonding|supramolecular assembly|Chemistry|QD1-999"
+
+This can be transformed into an array using the ```split``` function specifying the pipe character ( | ) as the separating character. Recall the cautionary note about separator choice from [Working with Data](https://librarycarpentry.org/lc-open-refine/03-working-with-data/index.html).
+.
 ```
-value.split(",")
+value.split("|")
 ```
-This would create the array containing the days of the week:
+This would create the array containing a list of subject headings, separated by a pipe character | (as in the first example above). In the transformation preview the array will display as a list of comma separated values in double quotes, with the whole array surrounded by square brackets.
 
-["Monday","Tuesday","Wednesday","Thursday","Friday","Saturday","Sunday"]
+This subject string can be found for the title "The crystal structures of three clozapinium salts: different molecular configurations, and supramolecular assembly in one, two and three dimensions" in the original project. 
 
-This can be combined with array operations like ```sort```. For example, assuming the cell contains the same value as above, then the function
+This can be combined with array operations like ```uniques```. For example, assuming the cell contains the same value as above, then the function
 ```
-value.split(",").sort()
+value.split("|").uniques()
 ```
-would result in an array containing the days of the week sorted in alphabetical order:
+"crystal structure", "clozapinium", "molecular configuration", "hydrogen bonding", "supramolecular assembly", "Chemistry", "QD1-999"
 
-["Friday","Monday","Saturday","Sunday","Thursday","Tuesday","Wednesday"]
+Compared to the first example, now the second 'crystal structure' has been removed. 
 
-To output a value from an array you can either select a specific value depending on its position in the list (with the first position treated as 'zero'). For example
+You can extract a specific item from the array using the square bracket notation and number for position in sequence:
 ```
-value.split(",")[0]
+value.split("|")[0]
 ```
-would extract the first value from the array created by the ```split``` function. In the above example this would be "Monday"
+["crystal structure"]
```

Comment:
Add line above to say "Would result in the string"
Remove square brackets and inverted commas as this is just a string